src/iuse_actor.cpp

```diff
@@ -4376,6 +4376,17 @@ int sew_advanced_actor::use( player &p, item &it, bool, const tripoint & ) const
                ( it.charges >= thread_needed && has_enough[mat_item] );
     };
 
+    const auto get_compare_color = [&](const int before, const int after, const bool higher_is_better) {
+        return before == after ? c_unset : (after > before == higher_is_better ? c_light_green : c_red);
```

Comment:
gcc warns parentheses error.
```suggestion
        return before == after ? c_unset : ((after > before) == higher_is_better ? c_light_green : c_red);
```